.gitignore

```diff
@@ -9,3 +9,4 @@ coverage
 node_modules
 npm-debug.log
 reports
+package-lock.json
```

Comment:
There are some other things I develop on that do use `package-lock`. I have been adding `package-lock=false` locally to a projects npmrc when the lockfile is not needed. DB might consider something like this since the default behavior of NPM is to include the package-lock. I would assume most outside devs will not have changed this setting in their root npmrc, so it may be worthwhile to add directly into each projects npmrc.